.kokoro/samples-test.sh

```diff
@@ -22,6 +22,12 @@ export NPM_CONFIG_PREFIX=${HOME}/.npm-global
 export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/secret_manager/long-door-651-kokoro-system-test-service-account
 export GCLOUD_PROJECT=long-door-651
 
+# For universe domain testing
+export TEST_UNIVERSE_DOMAIN_CREDENTIAL=$(realpath ${KOKORO_GFILE_DIR}/secret_manager/client-library-test-universe-domain-credential)
+export TEST_UNIVERSE_DOMAIN=$(gcloud secrets versions access latest --project cloud-devrel-kokoro-resources --secret=client-library-test-universe-domain)
```

Comment:
Are we sure this is the right secret name? Looks like the test is failing because this envvar isn't set.

Reply:
Due to the previous test failure, I've identified and resolved the issue by moving the code to the appropriate file. And now tests are passing successfully.